src/OpenTK/Math/Vector2.cs

```diff
@@ -1083,9 +1083,8 @@ public static void Transform(ref Vector2 vec, ref Quaternion quat, out Vector2 r
         /// <returns>Result of the division.</returns>
         public static Vector2 operator /(Vector2 vec, float scale)
         {
-            float mult = 1.0f / scale;
-            vec.X *= mult;
-            vec.Y *= mult;
+            vec.X /= scale;
+            vec.Y /= scale;
             return vec;
```

Comment:
The idea behind the old version was probably to save a repeat divide. I reckon the JIT will probably take care of it and produce some faster format, but it would be interesting to see a profiled difference here, if any.